src/reducers/tags.js

```diff
@@ -48,7 +49,29 @@ const decrementForRemovedTag = (state, { type, error, meta }) => {
 
 export const tagsByQuery = composeReducers(
   appendPayloadByPath(FETCH_TAGS, 'meta.cache.id', 'items', t => t.id),
-  filterOutRemovedTag
+  filterOutRemovedTag,
+  (state, { type, meta }) => {
```

Comment:
@levity - this is what I was asking about. I'm not sure why we have tagsByCommunity and tagsByQuery both containing tag data. makes sense to pattern it after posts, and have tagsByCommunity contain the data and tagsByQuery just the ids.

But I didn't want to go further down that direction without checking with you.

I can also just leave it as is if you think there's higher priorities.